satpy/readers/hdfeos_base.py

```diff
@@ -215,14 +214,40 @@ def load_dataset(self, dataset_name, is_category=False):
         from satpy.readers.hdf4_utils import from_sds
 
         dataset = self._read_dataset_in_file(dataset_name)
-        dask_arr = from_sds(dataset, chunks=CHUNK_SIZE)
+        chunks = self._chunks_for_variable(dataset)
+        dask_arr = from_sds(dataset, chunks=chunks)
```

Comment:
I forgot again, can we get the storage chunking from pyhdf?

Reply:
No. Not with pyhdf. It doesn't have any wrappers around the C API for chunk related stuff.